app/uk/gov/hmrc/transitmovementsrouter/controllers/MessagesController.scala

```diff
@@ -110,6 +110,7 @@ class MessagesController @Inject() (
         persistenceResponse <- persistenceConnector
           .postObjectStoreUri(movementId, messageId, messageType, ObjectStoreURI(objectSummary.location.asUri))
           .asPresentation
+        _ = pushNotificationsConnector
```

Comment:
You're not actually calling the function here

`_ = pushNotificationsConnector.post(movementId, persistenceResponse.messageId, None).asPresentation`